File: Solution/ui/reactapp/src/components/assessment/membership/entities/Member.ts
interface Member {
  id: number;
  name: string;
  email: string;
  role: "admin" | "mentor" | "sme" | "employer" | "student";
  status: "active" | "inactive" | "suspended" | "pending";
  joinDate: string;
  lastLoginDate: string;
  department: string;
}
export default Member;

File: Solution/Backend/java/webapi/src/main/java/com/transflower/tflcomentor/fileio/Entity/Member.java
package com.transflower.tflcomentor.fileio.Entity;

import java.time.LocalDate;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder

public class Member {
    private int id;
    private String name;
    private String email;
    private String role;
    private String status;
    private LocalDate joinDate;
    private LocalDate lastLoginDate;
    private String department;
  
}
